frontend/src/modules/WellLogViewer/settings/components/ProviderManagerComponentWrapper.tsx

```diff
@@ -0,0 +1,328 @@
+import React from "react";
+
+import { HorizontalRule, MultilineChart, ShowChart, ViewDay } from "@mui/icons-material";
+import { useQueryClient } from "@tanstack/react-query";
+import { useAtom } from "jotai";
+import _ from "lodash";
```

Comment:
Replace import _ from "lodash"; with import of correct utils, as done for rest of the module code in WellLogViewer